oauth2_http/java/com/google/auth/oauth2/JwtClaims.java

```diff
@@ -68,10 +73,10 @@
    *
    * @return additional claims
    */
-  abstract Map<String, String> getAdditionalClaims();
+  abstract Map<String, ?> getAdditionalClaims();
```

Comment:
this is a breaking change. It should be possible to keep the old one marked as Deprecated and calling this new signature internally